openreview/arr/helpers.py

```diff
@@ -1034,6 +1035,25 @@ def __init__(self, client_v2, venue, configuration_note, request_form_id, suppor
                     invitation=stat_inv
                 )
 
+            if not openreview.tools.get_invitation(self.client_v2, f"{role}/-/Emergency_Score"): # Hold "Requested" or "Reassigned", head=submission ID
+                m._create_edge_invitation(f"{role}/-/Emergency_Score")
```

Comment:
@haroldrubio do we need to add the Emergency_Score to the edge browser urls?

Reply:
Yes I'll be changing the links in my webfield branch

Reply:
Oh I see I'll update the links in my branch after this is merged